packages/integration-tests/src/api/experience-api/password-verification.ts

```diff
@@ -0,0 +1,20 @@
+import { type PasswordVerificationPayload } from '@logto/schemas';
+
+import api from '../api.js';
+
+import { experienceVerificationApiRoutesPrefix } from './const.js';
+
+type VerificationResponse = {
+  verificationId: string;
+};
+
+export const createPasswordVerification = async (
```

Comment:
since this looks coupled with experience, will it be better to create an experience test class thus no need for explicitly maintaining and passing the cookie?